flang/lib/Lower/OpenMP.cpp

```diff
@@ -11,6 +11,8 @@
 //===----------------------------------------------------------------------===//
 
 #include "flang/Lower/OpenMP.h"
+#include "SymbolMap.h"
```

Comment:
Is this include required? (I probably just missed it while skimming...)

Reply:
Thanks! Yea, not needed. Must be residual of my other work.